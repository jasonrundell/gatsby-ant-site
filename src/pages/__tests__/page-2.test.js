import React from 'react'
import renderer from 'react-test-renderer'

import SecondPage from '../page-2'

describe('SecondPage', () => {
  it('renders correctly', () => {
    const data = {
      site: {
        siteMetadata: {
          title: "SecondPage",
        },
      },
    }

    const tree = renderer.create(<SecondPage data={data} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})