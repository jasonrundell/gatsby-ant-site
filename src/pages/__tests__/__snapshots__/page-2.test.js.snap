// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondPage renders correctly 1`] = `
Array [
  <header
    style={
      Object {
        "background": "rebeccapurple",
        "marginBottom": "1.45rem",
      }
    }
  >
    <div
      style={
        Object {
          "margin": "0 auto",
          "maxWidth": 960,
          "padding": "1.45rem 1.0875rem",
        }
      }
    >
      <h1
        style={
          Object {
            "margin": 0,
          }
        }
      >
        <a
          href="/"
          style={
            Object {
              "color": "white",
              "textDecoration": "none",
            }
          }
        >
          SecondPage
        </a>
      </h1>
    </div>
  </header>,
  <div
    className="container"
  >
    <main>
      <h1>
        Hi from the second page
      </h1>
      <p>
        Welcome to page 2
      </p>
      <a
        href="/"
      >
        Go back to the homepage
      </a>
    </main>
    <footer>
      Â© 
      2019
      , Built with
       
      <a
        href="https://www.gatsbyjs.org"
      >
        Gatsby
      </a>
    </footer>
  </div>,
]
`;
